website/conferences/message.py

```diff
@@ -164,3 +160,12 @@ def attachments(self):
                 range(count),
             ),
         )
+
+    @property
+    def find_allowed_types(self):
```

Comment:
Since this is a property rather than a method, `allowed_types` would be a more appropriate name.